guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -45,6 +45,9 @@ ifdef::satellite[]
 ----
 # subscription-manager repos --enable \
 {RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+
+# subscription-manager repos --enable \
+{RepoRHEL8ServerSatelliteServerProductVersion}
```

Comment:
The step here reads "Ensure that the {Project} Maintenance repository is enabled:", but we're enabling the non-Maintenance repository here, to workaround a bug.

I'd make it a separate step, called ". Ensure that the {Project} {ProjectVersion} repository is enabled:", maybe even add a comment that this is a temporary workaround for a BZ.

Reply:
Addressed :+1: